docs/tutorial/source-plugin-tutorial.md

```diff
@@ -110,7 +110,7 @@ _You can include the plugin by using its name if you are using [npm link or yarn
 
 You can now navigate into the `example-site` folder and run `gatsby develop`. You should see a line in the output in the terminal that shows your plugin loaded:
 
-```shell
+```shell:title=example-site
```

Comment:
I wonder if this can be done for every shell command on the Gatsby website? I think not.  I would rather leave a shell tag without a title because the text before it implies running it in the `example-site` directory. 
```suggestion
```shell
```

Reply:
if you look into the full tutorial text: the shell commands in this tutorial use this title to mark the directory if it is in a subdirectory - only this 2 places was not. 

to be consistent in one doc there are 2 solutions: remove it from the others, or add it here.

i would add it here because that helps a little the reader - i liked here that also the js code examples had the subdirs are in the title :)

Reply:
I think kyle was trying to be more specific because it wasn't implied in the two places the :title=example-site tag was used with shell. The other 4 places were implied.

Reply:
https://github.com/gatsbyjs/gatsby/blob/26307583863343e84c73e18f14d3d04717833893/docs/tutorial/source-plugin-tutorial.md#L437-L439
https://github.com/gatsbyjs/gatsby/blob/26307583863343e84c73e18f14d3d04717833893/docs/tutorial/source-plugin-tutorial.md#L532-L534

in the other 2 cases it is noted also the line before. 

but i think, more easy for the reader is if he reads in which dir he should execute the commands